### PR TITLE
LPD-18697 Storefront Returns listing page

### PR DESCRIPTION
**Pull request by gianmarcobrunialti**
https://liferay.atlassian.net/browse/LPD-18697

~~There's a dependency on https://github.com/liferay-commerce/liferay-portal/pull/4774, which has been already forwarded to https://github.com/brianchandotcom/liferay-portal/pull/148452~~

Sent here: https://github.com/liferay-commerce/liferay-portal/pull/4823

### CODE REVIEW

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

**Comment by gianmarcobrunialti** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json [hunk 1]
```diff
@@ -249,6 +249,27 @@
...
 	},
+	{
+		"icon": "icon_orders.png",
+		"layoutTemplateId": "1_column",
+		"name": "Returns",
+		"permissions": [
+			{
+				"actionIds": [
+				],
+				"roleName": "Guest",
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json:260]: Guest will never be able to place returns I guess, does it makes sense to make the page available to it?

Comment (gianmarcobrunialti) [reply modules/apps/commerce/commerce-theme-minium/commerce-theme-minium-site-initializer/src/main/resources/com/liferay/commerce/theme/minium/site/initializer/internal/dependencies/layouts.json]: Removed, also as per https://liferay.slack.com/archives/G9CMBBA9H/p1711370833030989?thread_ts=1711365042.099939&cid=G9CMBBA9H 👍🏼

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:test:sf

**Comment by luca-pellizzon** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/application/list/CommerceReturnPanelApp.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+	@Override
+	public String getPortletId() {
+		return CommercePortletKeys.COMMERCE_RETURN;
+	}
+
+	@Override
+	public boolean isShow(PermissionChecker permissionChecker, Group group)
+		throws PortalException {
+
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-9361")) {
```
Comment (CrescenzoRegaLF) [inline modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/application/list/CommerceReturnPanelApp.java:47]: @gianmarcobrunialti you should train to the new feature flag, see [here](https://github.com/liferay-commerce/liferay-portal/pull/4801)

cc @luca-pellizzon

**Pull request closed without merging**